spec/adapters/datatables_spec.rb

```diff
@@ -14,8 +16,22 @@
   end
 
   describe "initialization Tables" do
-    it "Table class must be Daru::DataTables::DataTable " do
+    it "Table class must be Daru::DataTables::DataTable" do
       expect(Daru::View::Table.new.table).to be_a Daru::DataTables::DataTable
     end
+
+    it "Table class must be Daru::DataTables::DataTable when data objects" \
+       " are of class Array" do
+      expect(Daru::View::Table.new(@string_array, @options).table).to be_a Daru::DataTables::DataTable
```

Comment:
I hope from next time I don't need to remind you.

```
expect(....).
  to eq(.....)
or

expect(....)
  .to eq(.....)
or

expect(....).to eq(
  .....
)
or

expect(
  ....
).to eq(.....)
or

expect(
  ....
).to eq(
  .....
)
```

Reply:
Sorry. I will keep it in my mind.